src/services/app/user.ts

```diff
@@ -494,4 +494,51 @@ export default class UserService {
         }
         return updated[1][0];
     }
+
+    public static async delete(address: string): Promise<boolean> {
+        try {
+            const manager = await this.manager.findOne({
+                where: { active: true, address },
+            });
+
+            if (manager) {
+                const managersByCommunity = await this.manager.findAll({
+                    where: {
+                        active: true,
+                        communityId: manager.communityId,
+                    },
+                    include: [
```

Comment:
Should we also consider `where: { deletedAt: null }` ?

Reply:
yes, I included this

Reply:
Can you add just one more test to test this case, please?

Reply:
Done.
I forgot to add in the authentication endpoint the option to "recover" the account, so I made this change and its respective tests